sdk/trace/trace_test.go

```diff
@@ -482,63 +482,30 @@ func TestEventsOverLimit(t *testing.T) {
 	}
 }
 
-func TestAddLinks(t *testing.T) {
-	te := &testExporter{}
-	tp, _ := NewProvider(WithSyncer(te))
-
-	span := startSpan(tp, "AddLinks")
-	k1v1 := key.New("key1").String("value1")
-	k2v2 := key.New("key2").String("value2")
-
-	sc1 := core.SpanContext{TraceID: core.TraceID([16]byte{1, 1}), SpanID: core.SpanID{3}}
-	sc2 := core.SpanContext{TraceID: core.TraceID([16]byte{1, 1}), SpanID: core.SpanID{3}}
-
-	link1 := apitrace.Link{SpanContext: sc1, Attributes: []core.KeyValue{k1v1}}
-	link2 := apitrace.Link{SpanContext: sc2, Attributes: []core.KeyValue{k2v2}}
-	span.AddLink(link1)
-	span.AddLink(link2)
-
-	got, err := endSpan(te, span)
-	if err != nil {
-		t.Fatal(err)
-	}
-
-	want := &export.SpanData{
-		SpanContext: core.SpanContext{
-			TraceID:    tid,
-			TraceFlags: 0x1,
-		},
-		ParentSpanID:    sid,
-		Name:            "AddLinks/span0",
-		HasRemoteParent: true,
-		Links: []apitrace.Link{
-			{SpanContext: sc1, Attributes: []core.KeyValue{k1v1}},
-			{SpanContext: sc2, Attributes: []core.KeyValue{k2v2}},
-		},
-		SpanKind: apitrace.SpanKindInternal,
-	}
-	if diff := cmpDiff(got, want); diff != "" {
-		t.Errorf("AddLink: -got +want %s", diff)
-	}
-}
-
 func TestLinks(t *testing.T) {
 	te := &testExporter{}
 	tp, _ := NewProvider(WithSyncer(te))
 
-	span := startSpan(tp, "Links")
 	k1v1 := key.New("key1").String("value1")
 	k2v2 := key.New("key2").String("value2")
 	k3v3 := key.New("key3").String("value3")
 
 	sc1 := core.SpanContext{TraceID: core.TraceID([16]byte{1, 1}), SpanID: core.SpanID{3}}
 	sc2 := core.SpanContext{TraceID: core.TraceID([16]byte{1, 1}), SpanID: core.SpanID{3}}
 
-	span.Link(sc1, key.New("key1").String("value1"))
-	span.Link(sc2,
+	span := startSpan(tp, "Links",
+		apitrace.LinkedTo(sc1, key.New("key1").String("value1")),
+		apitrace.LinkedTo(sc2,
+			key.New("key2").String("value2"),
+			key.New("key3").String("value3"),
+		),
+	)
+	apitrace.LinkedTo(sc1, key.New("key1").String("value1"))
+	apitrace.LinkedTo(sc2,
```

Comment:
These two `LinkedTo` are not being used, just remove them.

Reply:
Sorry, Got it. I already put them in ```startSpan()``` function.